src/composables/feed/useDeleteById.ts

```diff
@@ -1,19 +1,25 @@
+import HttpMethod from '@/enum/http-method.enum'
 import client from '@/services/client.service'
+import { Ref, ref } from 'vue'
 
-function useDeleteById(){
-  const deletePostById = async (id: number) => {
-    await client({
-      method: 'DELETE',
+function useDeleteById(id: number): { res: Ref<boolean> } {
```

Comment:
is there another way for us to delete the item aside from the id? If none, you can just rename your function to useDeletePost.

Reply:
rename function to use-delete-Post.